### ISSUE TITLE
Re-import 2024 APPG and Select Committee memberships, once they’re re-convened after Summer recess

### ISSUE THREAD

**Issue by zarino** [mysociety/local-intelligence-hub#595]
APPGs and Select Committees were suspended during the election period, and haven’t re-started yet.

Parliament says the register of APPGs, at least, will be ["published in August 2024"](https://www.parliament.uk/mps-lords-and-offices/standards-and-financial-interests/parliamentary-commissioner-for-standards/registers-of-interests/register-of-all-party-party-parliamentary-groups/), but realistically, the data will likely be patchy until the _next_ release after that, once MPs have returned from the Summer recess ([30th July – 2nd September](https://www.parliament.uk/about/faqs/house-of-commons-faqs/business-faq-page/recess-dates/)).

Or, given there’s only 10 days between the end of recess, and the start of party conference season (12th September), there’s always the chance that we might have to wait until 7th October or later for the best data.

**Comment by zarino** [comment]
Related: #535

**Comment by struan** [comment]
I think the conclusion here is that we are waiting till the 8th of October for this data to be useful?

**Comment by zarino** [comment]
Yes, we’ve done without the APPGs for a while now, I don’t think there’s any harm in waiting a little longer (potentially until WhoFundsThem is complete) to do it properly and all in one go.

**Comment by zarino** [comment]
Closing in favour of #641 (for APPGs) and #649 (for Select Committees).
